content/docs/reference/cli-quickstart.md

```diff
@@ -0,0 +1,226 @@
+
+## Authenticate
+
+Authenticate with your Neon account using one of these methods:
+
+**Web Authentication (recommended)**
+
+Run the command below to authenticate through your browser:
+
+```bash
+neon auth
+```
+
+This will open a browser window where you can authorize the CLI to access your Neon account.
+
+**API Key Authentication**
+
+Alternatively, you can use a Neon API key. You can create one in the Neon Console. See [Create a personal API key](https://neon.tech/docs/manage/api-keys#create-a-personal-api-key).
+
+```bash
+neon projects list --api-key <your-api-key>
+```
+
+To avoid entering your API key with each command, you can set it as an environment variable:
+
+```bash
+export NEON_API_KEY=<your-api-key>
+```
+
+For more about authenticating, see [Neon CLI commands — auth](/docs/reference/cli-auth).
+
+## Set up your context file
+
+Context files allow you to use CLI commands without specifying your project or organization IDs every time.
+
+To set the context for your Neon project:
+
+```bash
+neon set-context --project-id <your-project-id>
+```
+
+You can find your Neon project ID by opening your project in the Neon Console and navigating to **Settings** > **General**.
+
+Th the `set-context` command creates a `.neon` file in your current directory with your project context.
+
+```bash
+$ cat .neon
+
+{
+  "projectId": "cool-darkness-12345678"
+}%                         
+```
+
+You can also create named context files for different projects:
+
+```bash
+neon set-context --project-id <your-project-id> --context-file my-project-context
+```
+
+For organization-level operations, you can create named organization context file:
+
+```bash
+neon set-context --org-id <your-org-id> --context-file my-org-context
+```
+
+To switch contexts, add the `--context-file` option to any command, specifying your context file:
+
+```bash
+neon branches list --context-file Documents/my-org-context
+```
+
+For more about the `set-context` command, see [Neon CLI commands — set-context](/docs/reference/cli-set-context). 
+
+## Enable shell completion
+
+Next, you can set up autocompletion to make using the CLI faster:
+
+<Tabs labels={["Bash", "Zsh"]}>
+
+<TabItem>
+
+```bash
+neon completion >> ~/.bashrc
+source ~/.bashrc
+```
+
+</TabItem>
+
+<TabItem>
+
+```bash
+neon completion >> ~/.zshrc
+source ~/.zshrc
+```
+
+</TabItem>
+
+</Tabs>
+
+Now you can press **Tab** to complete commands and options. For further details, see [Neon CLI commands — completion](/docs/reference/cli-completion).
+
+## Common operations
+
+Here are some common operations you can perform with the Neon CLI:
+
+### List your projects
+
+```bash
+neon projects list
+```
+
+If you want to list projects in your organization, don't forget to set your organization context or specify your `--org-id <your-org-id>`. Otherwise, you'll list the projects in your personal Neon account.
```

Comment:
```suggestion
If you want to list projects in your organization, don't forget to set your organization context or specify `--org-id <your-org-id>`. Otherwise, you'll list the projects in your personal Neon account.
```